repos/system_upgrade/el7toel8/libraries/dnfplugin.py

```diff
@@ -89,7 +89,7 @@ def backup_debug_data(context):
             api.current_logger().warn('Failed to copy debugdata. Message: {}'.format(str(e)), exc_info=True)
 
 
-def _transaction(context, stage, target_repoids, tasks, test=False):
+def _transaction(context, stage, target_repoids, tasks, test=False, cmd_prefix=False):
```

Comment:
just a nitpick, you should use empty list instead of False as a default for cmd_prefix if you are expecting lists there

Reply:
makes sense, fixed...

Reply:
dangerous-default-value :) 

"None" it is then...